backend/core/test/listings/listings.e2e-spec.ts

```diff
@@ -32,33 +32,60 @@ describe("Listings", () => {
 
   it("should return all listings", async () => {
     const res = await supertest(app.getHttpServer()).get("/listings").expect(200)
-    expect(res.body.map((listing) => listing.id).length).toBeGreaterThan(0)
+    expect(res.body.items.map((listing) => listing.id).length).toBeGreaterThan(0)
   })
 
+  it("should return the first page of paginated listings", async () => {
+    // Make the limit 1 less than the full number of listings, so that the first page contains all
+    // but the last listing.
+    const page = "1"
+    const limit = (allSeeds.length - 1).toString()
+    const params = "/?page=" + page + "&limit=" + limit
+    const res = await supertest(app.getHttpServer())
+      .get("/listings" + params)
+      .expect(200)
+    expect(res.body.items.length).toEqual(allSeeds.length - 1)
+  })
+
+  it("should return the last page of paginated listings", async () => {
+    // Make the limit 1 less than the full number of listings, so that the second page contains
+    // only one listing.
+    const page = "2"
+    const limit = (allSeeds.length - 1).toString()
```

Comment:
In our fork, `allSeeds` still existed when we wrote this test. What should we use now to get the list of seeded listings?

Reply:
@avaleske There is still a `allSeeds` variable but it's hidden inside `backend/core/src/seed.ts:seedListings`, feel free to expose it through a function

Reply:
Done, thanks!